### PR TITLE
added overload for is_ongoing field and added getter for is_ongoing

### PR DESCRIPTION
**Pull request by carycheng**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @carycheng, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: doc/legal_holds.md [hunk 1]
```diff
@@ -68,8 +68,15 @@ method will let you create a new legal hold policy with a specified name, descri
...
 
+If you wish to create an ongoing Legal Hold Policy with no end date, call [`createOngoing(BoxAPIConnection api, String name, String description)`][create-ongoing].
+
+```java
+BoxLegalHoldPolicy.Info policyInfo = BoxLegalHoldPolicy.createOngoing(api, name, description);
+```
+
 [create-new-legal-hold-policy]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxLegalHoldPolicy.html#create-com.box.sdk.BoxAPIConnection-java.lang.String-
 [create-new-legal-hold-policy-with-dates]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxLegalHoldPolicy.html#create-com.box.sdk.BoxAPIConnection-java.lang.String-java.lang.String-java.util.Date-java.util.Date-
+[create-ongoing]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxLegalHoldPolicy.html#createOngoing-com.box.sdk.BoxAPIConnection-java.lang.String-java.lang.String
```
Comment (mattwiller) [inline doc/legal_holds.md]:
```suggestion
[create-ongoing]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxLegalHoldPolicy.html#createOngoing-com.box.sdk.BoxAPIConnection-java.lang.String-java.lang.String-
```

File: src/main/java/com/box/sdk/BoxLegalHoldPolicy.java [hunk 2]
```diff
@@ -330,6 +353,11 @@ public void updateInfo(BoxLegalHoldPolicy.Info info) {
          */
         private String releaseNotes;
 
+        /**
+         * @see #getIsOngoing()
+         */
+        private Boolean isOngoing;
```
Comment (mattwiller) [inline src/main/java/com/box/sdk/BoxLegalHoldPolicy.java]: We usually use the primitive type for these things; how does it work using the class?

File: src/test/java/com/box/sdk/BoxLegalHoldPolicyTest.java [hunk 3]
```diff
@@ -110,6 +110,40 @@ public void testCreateNewLegalHoldPolicySucceedsAndSendsCorrectJson() throws IOE
...
+
+        result = TestConfig.getFixture("BoxLegalHold/PostOngoingLegalHoldPolicies201");
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.post(WireMock.urlPathEqualTo(legalHoldsURL))
+                .withRequestBody(WireMock.equalToJson(policyObject.toString()))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(result)));
+
+        BoxLegalHoldPolicy.Info policyInfo = BoxLegalHoldPolicy.createOngoing(this.api, policyName, null);
```
Comment (mattwiller) [inline src/test/java/com/box/sdk/BoxLegalHoldPolicyTest.java]: You should add a description here for better code coverage

**Pull request merged**